automation-scripts/brandColor.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs').promises;
```

Comment:
Can we add comments at the top of these files explaining that they aren't used in the build steps and that there is still some manual steps to transpile the JSON to JS tokens